pageserver/src/repository.rs

```diff
@@ -86,6 +86,31 @@ pub trait Timeline: Send + Sync {
     /// NOTE: This has nothing to do with checkpoint in PostgreSQL. We don't
     /// know anything about them here in the repository.
     fn checkpoint(&self) -> Result<()>;
+
+    /// Snapshot of a timeline. Includes history up until lsn().
```

Comment:
Yep, start from the beginning of time. At the time this function is called, we read the last valid LSN and only send updates up to and including it. (It can be seen via the `lsn()` function on `Snapshot`)

Will clarify the comment